### DISCUSSION TITLE
Vercel OG - Font style bold not working with custom .woff file

### DISCUSSION THREAD

**Discussion by ShubhamVerma1811** [vercel/next.js#43794] [Help]
Hey there, I have a single `Karla.woff` file in public directly and importing it and using the `font-bold` with tailwind doesn't seem to work.

I am not sure what I am missing here.

Here's the actual code used in the production
https://github.com/ShubhamVerma1811/Website/blob/main/src/pages/api/og.tsx

Here is the sample code

```jsx
import { ImageResponse } from '@vercel/og';
import { NextRequest } from 'next/server';
import React from 'react';

export const config = {
  runtime: 'experimental-edge',
};

const font = fetch(
  new URL('../../../public/assets/fonts/Karla/Karla.woff', import.meta.url)
).then((res) => res.arrayBuffer());

export default async function handler(req: NextRequest) {
  const fontData = await font;

  return new ImageResponse(
    (
      <div tw='h-full w-full flex flex-col bg-gray-200 items-center relative'>
        <h1
          tw='text-6xl font-bold mt-30 mx-20 text-center'
          style={{
            fontWeight: 700,
          }}>
          "TITLE HERE"
        </h1>
      </div>
    ),
    {
      width: 1600,
      height: 840,
      fonts: [
        {
          name: 'Karla',
          data: fontData,
          style: 'normal',
        },
      ],
    }
  );
}

```

**Comment by ShubhamVerma1811** [comment]
Hey @shuding,
I avoid tagging people unless absolutely necessary.
Can you please help me with this?

**Reply by shuding** [reply]
Hey! Is the problem that it doesn't render with the font you specified? I don't see `font-family: Karla` anywhere though.

**Reply by ShubhamVerma1811** [reply]
It renders the font that I've specified i.e Karla.

But it doesn't apply font style of bold when I am using `font-bold` class.


Here is the example

https://shubhamverma.me/api/og?title=I%20should%27ve%20been%20bold
